Src/HelloWorld.java

```diff
@@ -4,6 +4,6 @@
 { 
 	public static void main(String args[]) 
 	{ 
-		System.out.println("Hello, World - Bangalore, KARNATAKA"); 
+		System.out.println("Hello, World - Bangalore, KA"); 
```

Comment:
This is the correct way to specify address

Reply:
Ok Thanks